orchestrator/src/test/java/com/yahoo/vespa/orchestrator/resources/HostResourceTest.java

```diff
@@ -61,7 +62,7 @@
  * @author hakonhall
  */
 public class HostResourceTest {
-    private static final Timer timer = mock(Timer.class);
+    private static final Clock clock = mock(Clock.class);
```

Comment:
No need to use a mock framework here. You could use e.g com.yahoo.test.ManualClock instead.

Reply:
Didn't know about the ManualClock.

Reply:
ManualClock would introduce an additional dependency here, and no manipulation of the clock is necessary - leaving as-is. Will move to ManualClock in TimeBudgetTestCase.java